packages/node/src/main.ts

```diff
@@ -16,16 +16,10 @@ initLogger(
 const { setProfiler } = require('@subql/node-core');
 setProfiler(argv.profiler);
 
-// Lazy import, to allow logger to be initialised before bootstrap()
-// As bootstrap runs services that requires logger
-const { bootstrap } = require('./init');
-if (
-  !(
-    argv._[0] === 'test' ||
-    argv._[0] === 'mmr-migrate' ||
-    argv._[0] === 'mmr-regen' ||
-    argv._[0] === 'force-clean'
-  )
-) {
+// Check for no subcommand
```

Comment:
do we need this too
```
(BigInt.prototype as any).toJSON = function () {
  return this.toString();
};
```

Reply:
I don't think so. Near doesn't seem to use much BigInt